### DISCUSSION TITLE
This is not a production server yet!

### DISCUSSION THREAD

**Discussion by DaSchTour** [nrwl/nx#8435] [General]
When generating a nest.js project you get this nice message in the `main.ts`
```
/**
 * This is not a production server yet!
 * This is only a minimal backend to get started.
 */
```

For me this somehow translates to: "Don't use this and we will not tell you how to use it, so stop using it". 

I think this is really scaring because there is no guidance on why it's not a production server and how to solve this. What steps do I need to do so I can safely remove this message?

**Comment by creotip** [comment]
If you want to run an app in production mode use the "production" flag, for example:
```shell
 nx run express-app:build:production 
```

**Reply by DaSchTour** [reply]
If that's the solution, why is there this comment in the main.ts? And why doesn't it simply state in the comment to run the build with production to get a production ready server? I'm not really convinced that this is the solution to this comment.
